apps/extension/src/routes/popup/home/inspect-address/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box } from '@penumbra-zone/ui/components/ui/box';
+import { IconInput } from '@penumbra-zone/ui/components/ui/icon-input';
+import { useEffect, useState } from 'react';
+import { MagnifyingGlassIcon } from '@radix-ui/react-icons';
+import { Result as TResult } from './types';
+import { getResultFromBech32mAddress } from './get-result-from-bech32m-address';
+import { Result } from './result';
+
+export const InspectAddress = () => {
```

Comment:
![image](https://github.com/penumbra-zone/web/assets/1121544/5e18778c-60c1-4872-841e-8ba94729331f)

Reply:
Could we add CSS that:
1 - Only causes the inspect box to expand downward and not cause the box to input to raise up on verification?
2 - Not affect the position of the account selector on verification expansion?